catalog-rest-service/src/main/resources/ui/src/components/EntityTable/EntityTable.component.tsx

```diff
@@ -516,9 +516,9 @@ const EntityTable = ({
                                     />
                                   </button>
                                 ) : (
-                                  <span className="tw-opacity-0 group-hover:tw-opacity-100">
+                                  <span className="tw-opacity-60 group-hover:tw-opacity-100 tw-text-grey-muted group-hover:tw-text-primary">
                                     <Tags
-                                      className="tw-border-main"
+                                      className=""
```

Comment:
if there is no class then we should remove this.

Reply:
My bad :)

Reply:
Done